pkg/action/cl001/ac001/explainer.go

```diff
@@ -2,8 +2,52 @@ package ac001
 
 import (
 	"context"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/giantswarm/microerror"
 )
 
 func (e *Explainer) explain(ctx context.Context) (string, error) {
-	return "", nil
+	crs, err := newCRs("https://g8s.codename.region.aws.gigantic.io:443")
```

Comment:
The host magic is a bit ugly but because of the necessary control plane connection in the explain commands we have to fake that a bit.